.nvmrc

```diff
@@ -1 +1 @@
-4.2.6
+6
```

Comment:
please do the same thing with this `.nvmrc` and use strings to denote the node versions in the `travis.yml` and this PR will be ready to merge too.

```yaml
node_js: 
  - "6"
  - "4"
```